zebrad/src/components/mempool.rs

```diff
@@ -364,17 +364,25 @@ fn remove_expired_transactions(
     tip_height: zebra_chain::block::Height,
 ) {
     let mut txid_set = HashSet::new();
+    // we need a separate set since we need the original unmined ID for reject()
```

Comment:
Confusing comment: we're rejecting by mined ID, but the comment says we need the unmined ID

```suggestion
    // we need a separate set, since reject() takes the original unmined ID,
    // then extracts the mined ID out of it
```